lib/version_utils.pm

```diff
@@ -230,14 +227,22 @@ sub is_microos {
     }
     elsif ($filter =~ /\d\.\d\+?$/) {
         # If we use '+' it means "this or newer", which includes tumbleweed
-        return ($filter =~ /\+$/) if ($version_is_tw && $type eq 'opensuse');
+        return ($filter =~ /\+$/) if $version_is_tw;
         return check_version($filter, $version, qr/\d{1,}\.\d/);
     }
     else {
         return $flavor eq $filter;    # Specific FLAVOR selector
     }
 }
 
+=head2 is_sle_micro
+Check if distribution is SUSE Linux Enterprise Micro
+=cut
```

Comment:
nitpicking but i believe the perlpod requires a space after  the `=head2` and maybe before the `=end` too. you can check the output http://podwebview.odyniec.net/

Reply:
done